### ISSUE TITLE
[react-admin-demo]: Some components don't use contrast color on dark theme.

### ISSUE THREAD

**Issue by WiXSL** [marmelab/react-admin#4180]
![dark-themej](https://user-images.githubusercontent.com/8268610/71023170-f09ff780-20e0-11ea-8449-fa88bd01f9f0.jpg)

**Comment by fzaninotto** [comment]
I'm not sure how this should be fixed. If the theme has a bad Primary color for the background (as it's the case here), then react-admin can't do anything.

Tweaking the primary color in the dark theme for the demo is the only thing we can do IMHO.

Thoughts?

**Comment by sidescraper** [comment]
i use 3.1.4, same problem here.

It also doesn't work in the DEMO
https://marmelab.com/react-admin-demo

**Comment by JulienMattiussi** [comment]
I changed contrast in demo to set dark theme more lisible.
https://github.com/marmelab/react-admin/pull/4399

Keep in mind that you have to customize theming of your app if the default Material-UI themes don't satisfy your needs.

```js
export const darkTheme = {
    palette: {
        primary: {
            main: '#7f90f0',
        },
        type: 'dark',
    },
};

<Admin
     {...props}
     theme={darkTheme}
/>
```

There is various way for this depending of your implementation
https://material-ui.com/customization/palette/
https://marmelab.com/react-admin/doc/3.2/Theming.html

**Comment by WiXSL** [comment]
Thanks you!

**Comment by fzaninotto** [comment]
Fixed by #4399
